labgrid/util/helper.py

```diff
@@ -89,20 +89,26 @@ def unregister(self, callback):
         assert callback in self.callbacks
         self.callbacks.remove(callback)
 
+    @staticmethod
+    def log_callback(message, process):
```

Comment:
This already introduces the process argument, should be moved to the third commit.

Reply:
You're absolutely right, I must've messed up the commits here.

Reply:
Rebased the PR and fixed this.